django_coverage_plugin/plugin.py

```diff
@@ -162,6 +163,15 @@ def file_tracer(self, filename):
     def file_reporter(self, filename):
         return FileReporter(filename)
 
+    def find_unexecuted_files(self, src_dir):
+        res = []
+        for i, (dirpath, dirnames, filenames) in enumerate(os.walk(src_dir)):
+            for filename in filenames:
+                if re.match(r"^[^.#~!$@%^&*()+=,]+\.html?$", filename):
```

Comment:
The complicated regex was derived from [here](https://github.com/nedbat/coveragepy/blob/master/coverage/files.py#L394), in the `find_python_files` function.

I tried to keep the code corresponding with the rest of the code base, but I agree that the suggestion you make is a lot more readable, and to me atleast; preferable.